src/base/attachment/resource/constant.ts

```diff
@@ -0,0 +1,8 @@
+export const ATTACHMENT = {
+  UNIT_LENGTH: {
+    ONE_KB: 1024,
+    ONE_MB: 1048576,
+    ONE_GB: 1073741824,
+  },
+  INVALID_SIZE_ERROR: "Nan size",
```

Comment:
I think it should be:
export const ONE_KB = 1024;
export const ONE_MB = ONE_KB ^ 2;
export const ONE_GB = ONE_KB ^ 3;
export const ATTACHMENT_INVALID_SIZE_ERROR = "NaN size";